cmd/zsysd/completion.go

```diff
@@ -11,18 +11,52 @@ import (
 func installCompletionCmd(rootCmd *cobra.Command) {
 	prog := rootCmd.Name()
 	var completionCmd = &cobra.Command{
-		Use:   "completion",
+		Use:   "completion [bash|zsh|powershell]",
 		Short: i18n.G("Generates bash completion scripts"),
```

Comment:
Needs, to change, it’s not only bash now! If we do autodetection (see below), we can write: "Generates completion scripts for your current shell"